lci-odoo10/odoo/addons/lci_ecole/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 lorenzo6ioti
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Lci Ecole',
+    'description': """
+        custom ecole""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'lorenzo6ioti',
+    'depends': ['ecole'
+    ],
+    'data':[
+
```

Comment:
remove blank line --> check with pep8/flake8 for global syntax of the source code